src/clj/puppetlabs/services/jruby/jruby_schemas.clj

```diff
@@ -99,19 +106,22 @@
                      (nil? (schema/check JRubyInstanceState @%)))
                'JRubyInstanceState))
 
+(def JRubyInstanceSchema
+  {:pool pool-queue-type
+   :id schema/Int
+   :max-requests schema/Int
+   :flush-instance-fn IFn
+   :state JRubyInstanceStateContainer
+   :scripting-container ScriptingContainer
+   schema/Keyword schema/Any})
```

Comment:
Did this end up being necessary because there's no way to use schema to check if something satisfies the `record`?  Or because the schema check for the record is too strict and doesn't tolerate the final `schema/Keyword schema/Any` thing?

As-is it seems like we'll need to be really diligent about making sure that the fields here match up with the fields in the Record if they change over time?  If so, and there's no way for us to build the schema off of the record definition itself, then maybe we could add a comment that mentions that you need to edit them simultaneously?  Dunno.